### DISCUSSION TITLE
老哥可以集成到脚本猫或者油猴上吗

### DISCUSSION THREAD

**Discussion by Agony6** [CXRunfree/Autovisor#12] [Q&A] [answered]
把脚本集成到脚本猫或者油猴上可以拿手机狐猴浏览器刷课方便些，希望老哥能早点在油猴或者脚本猫发一个，现在油猴脚本猫上没有好用的智慧树脚本了，求了

**Comment by CXRunfree** [comment, marked as answer]
正是因为智慧树全面封杀油猴类(JS)脚本，我才用Python写这个的，就算能做出来迟早也会失效，前端语言有这个弊端

**Reply by Xia0-0** [reply]
一劳永逸的是不可能的，所以要经常完善呀🤣🤣🤣
